cli/kv.go

```diff
@@ -119,23 +115,19 @@ func runPut(cmd *cobra.Command, args []string) {
 		return
 	}
 
-	count := len(args) / 2
-
-	keys := make([]string, 0, count)
-	values := make([]string, 0, count)
+	var b client.Batch
 	for i := 0; i < len(args); i += 2 {
-		keys = append(keys, unquoteArg(args[i], true /* disallow system keys */))
-		values = append(values, unquoteArg(args[i+1], false))
+		b.Put(
+			unquoteArg(args[i], true /* disallow system keys */),
+			unquoteArg(args[i+1], false),
+		)
```

Comment:
That's some unusual formatting. Any reason to put first arg on a newline and the closing paren on a line by itself?